blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -295,8 +303,10 @@ public void run() {
                     TimeUnit.MINUTES.toMillis(savePeriodMins));
         }
 
-        // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
-        new RestartGauge(Metrics.getRegistry(), RollupService.class);
+        if (shouldInstantiateRestartGauge) {
```

Comment:
Nope. I tried extracting a method called `initializeInstrumentationReporting` that returned the `RestartGuage` object, and then mocking that method with PowerMock, but it's getting really difficult. I think it's ok to leave the flag as-is for now, until we can refactor the class enough that we don't need it anymore.